src/main/java/me/botsko/prism/Prism.java

```diff
@@ -712,6 +749,45 @@ public void onDisable() {
             prismDataSource.dispose();
         }
         log("Closing plugin.");
+        for (Handler handler : prismLog.getHandlers()) {
+            handler.close();
+        }
         super.onDisable();
     }
+
+    private static class PrismFileHandler extends FileHandler{
+
+        public PrismFileHandler(File file) throws IOException, SecurityException {
+            super(file.toString());
+            setFormatter(new SimpleFormatter(){
+                @Override
+                public synchronized String format(LogRecord lr) {
+                    boolean mt = Bukkit.isPrimaryThread();
+                    String thread;
+                    if(mt){
```

Comment:
Codacy found an issue: [WhitespaceAround: 'if' is not followed by whitespace. Empty blocks may only be represented as {} when not part of a multi-block statement (4.1.3)](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5911179)